src/template/submission_guidelines.md

```diff
@@ -0,0 +1,6 @@
+# Submission Guidelines
+
+The following guidelines must be met prior to submission of your notebook:
```

Comment:
Also, please confirm this part with Jong

Reply:
I also setup issue #7 for this.